hunt/app/views/animals/edit.html.erb

```diff
@@ -1,2 +1,13 @@
-<h1>Animals#edit</h1>
-<p>Find me in app/views/animals/edit.html.erb</p>
+<div class="container">
+  <div class="row">
+    <div class="col">
+      <div class="card">
+        <div class="card-header"><%= 'Edit ' + @animal.name %></div>
+        <div class="card-body">
+          <%= render partial: "form", locals: {post: @animal, cancel_url: animal_path} %>
```

Comment:
It doesn't look like we're using this `post` variable anywhere since we're just accessing `@animal` in the partial, so this can be removed from this file and `new.html.erb`